packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -238,6 +240,18 @@ async function moveTeam({
     teamSlug: newSlug,
     orgSlug: org.slug || (orgMetadata?.requestedSlug ?? null),
   });
+
+  // Cancel existing stripe subscriptions once the team is migrated
+  const parsedMetadata = teamMetadataSchema.safeParse(team.metadata);
+  if (parsedMetadata.success) {
+    const subscriptionId = parsedMetadata.data?.subscriptionId;
+    if (subscriptionId) {
+      log.debug("Canceling stripe subscription", safeStringify({ team, subscriptionId }));
+      await stripe.subscriptions.cancel(subscriptionId);
```

Comment:
It can throw errors it seems atleast as per ChatGPT. We should handle it gracefully.
<img width="995" alt="Screenshot 2024-06-27 at 2 37 53 PM" src="https://github.com/calcom/cal.com/assets/1780212/1e63294a-7cd6-481d-bb76-e4742487cbd3">

Reply:
Sure one sec!